hudi-flink/src/main/java/org/apache/hudi/operator/StreamWriteOperatorCoordinator.java

```diff
@@ -153,6 +152,14 @@ public void checkpointCoordinator(long checkpointId, CompletableFuture<byte[]> r
   public void checkpointComplete(long checkpointId) {
     // start to commit the instant.
     checkAndCommitWithRetry();
+    // start new instant.
+    startInstant();
+  }
+
+  private void startInstant() {
+    this.inFlightInstant = this.writeClient.startCommit();
+    this.writeClient.transitionRequestedToInflight(conf.getString(FlinkOptions.TABLE_TYPE), this.inFlightInstant);
+    LOG.info("Create instant [{}]", this.inFlightInstant);
```

Comment:
Provide more information?

Reply:
What information

Reply:
For what table? For what table type? Actually, at the log point, the instant has been transitioned from `requested` to `inflight`, right? Not only been created.